lib/Mojo/mysql/PubSub.pm

```diff
@@ -137,10 +137,13 @@ sub _query_with_retry {
     # first line to avoid potential spurious matches if the error
     # e.g. contains a stack trace.
     my $err = $@;                                        # avoid stringifying $@ ...
-    croak $@ unless $err =~ /^\V*(?:retry|timeout)/i;    # ... and maybe rethrow it
-
-    # If we got here, we are retrying the query:
-    warn qq|[PubSub] (@{[$db->pid]}) retry ($sql) !!! $err\n| if DEBUG;
+    if ($err =~ /^\V*(?:retry|timeout|try restart)/i) {  # ... and maybe rethrow it
+      # If we got here, we are retrying the query:
+      warn qq|[PubSub] (@{[$db->pid]}) retry ($sql) !!! $err\n| if DEBUG;
+    } else {
```

Comment:
This PR does not conform with the codestyle. Please run perltidy and force push the changes.